torch/linalg/__init__.py

```diff
@@ -871,7 +871,8 @@
 """ + r"""
 
 Args:
-    A (Tensor): tensor of shape (*, n, n) where * is zero or more batch dimensions consisting of symmetric or Hermitian matrices.
+    A (Tensor): tensor of shape `(*, n, n)` where `*` is zero or more batch dimensions
+                    consisting of symmetric or Hermitian matrices.
                     `(*, n, n)` where `*` is one or more batch dimensions.
```

Comment:
Feel free to delete this line as well, as it's repeated. Same in the other function.

Reply:
Also, this line has extra indentation. Same in the other one